src/screens/Me/Me.js

```diff
@@ -51,18 +53,19 @@ class MeScreen extends React.Component<Props, State> {
   };
 
   validateWalletConnectQRCode = (uri: string) => {
```

Comment:
Since now it validates both Wallet Connect and Pillarwallet deep links maybe we can rename `validateWalletConnectQRCode` as for example `validateScannedQRCode`, what do you think?

Reply:
Sounds good, changed.